src/core/index.js

```diff
@@ -275,6 +275,43 @@ util.extend( corefn, {
     return this._private.container;
   },
 
+  mount: function( container ){
+    let cy = this;
+    let _p = cy._private;
+    let options = _p.options;
+
+    if( container == null ){ return; }
+
+    if( !is.htmlElement( container ) && is.htmlElement( container[0] ) ){
+      container = container[0];
+    }
+  
+    cy.stopAnimationLoop();
+    cy.destroyRenderer();
+
+    options.container = container;
+    _p.container = container;
+
+    options.styleEnabled = true;
+    _p.styleEnabled = true;
+
+    options.renderer.name = 'canvas';
```

Comment:
We can't really assume this generally.  The dev can specify which renderer to use at init, so we also probably need to have a `rendererOptions` argument for `options.renderer`.

If unspecified, it should use the canvas renderer like you have here.